src/clarity/checkboxes/_checkbox.clarity.scss

```diff
@@ -0,0 +1,19 @@
+clr-checkbox {
+  label.indeterminate {
+
+    background-color: $clr-form-field-background-color;
+    border: none;
+
+    &:before {
+      width: 75px;
+      height: 75px;
+      border: 1px solid darkblue;
+    }
+
+    &:after {
+      width: 50px;
+      height: 50px;
+      border: 1px solid darkorange;
+    }
```

Comment:
would it be a good idea to have standard sizes in $clr? or % type of sizes instead maybe?